worksheet/index.qmd

```diff
@@ -0,0 +1,137 @@
+---
+format: typst
+---
+
+```{=typst}
+#set page(
+  flipped: true, 
+  columns: 2, 
+  margin: (x: 0.5in, y: 1in),
+  numbering: none
+)
+
+#set list(marker: ([#sym.square.stroked], [-]))
+```
+
+## Writing your first documentation PR
+
+The goal of this worksheet is to guide you through the steps needed to successfully create a documentation PR (pull request). Empty spaces have been placed on this sheet for your notes.
+
+### 0. Get ready
+
+- `install.packages("pak")`
+
+- `pak::pak("devtools")`
+
+- .Set up your GitHub personal access token.
+    - `usethis.r-lib.org` -> Articles -> Managing Git(Hub) Credentials
+
+- Call `usethis::git_sitrep()` and check that:
+    - Your name and email address appears in "Git config (global)"
+    - Your GitHub user name is found under "Github"
+
+```{=typst}
+#v(1.5in)
+```
+
+### 1. Find and claim an issue
+
+Browse the Post-Its, looking for something of interest. If you've never done a PR before, we recommend that you start with a documentation issue since there are slightly fewer moving parts. We also encouraged you to team up with a (new) friend and tackle something together!
+
+Once you've picked a Post-it, open the issue and read the details in full. At this point, you might discover the issue is outside your wheelhouse; if so, no problem, just return the Post-it to the wall and try again.
+
+```{=typst}
+#colbreak()
+```
+
+```{=typst}
+#align(center)[
```

Comment:
Nice! Just a note that when I printed this from my mac, it was set to auto-scale the page, which reduced it to 96% for some reason.

Reply:
i hope you are suggesting that we figure out how to print without autoscale instead of setting the dimensions to 3.125 because that just seems wild 😅